app/components/candidate_interface/degrees_review_component.rb

```diff
@@ -38,12 +38,16 @@ def title(degree)
 
     def degree_type_with_honours(degree)
       if degree.grade&.include? 'honours'
-        "#{degree.qualification_type} (Hons)"
+        "#{abbreviate_degree(degree.qualification_type)} (Hons)"
       else
-        degree.qualification_type
+        abbreviate_degree(degree.qualification_type)
       end
     end
 
+    def abbreviate_degree(name)
+      Hesa::DegreeType.find_by_name(name)&.abbreviation || name
```

Comment:
👍 looks like there are a few HESA degree types without abbreviations but this handles that situation neatly